### PR TITLE
Addition of new test utility to get VCAP credentials.

### PR DESCRIPTION
**Pull request by nwspeete-ibm**

### CODE REVIEW

**Comment by nwspeete-ibm** [comment]
This pull request completes the discussion from https://github.com/openwhisk/openwhisk/pull/443

File: tests/src/common/TestUtils.java [hunk 1]
```diff
@@ -103,6 +108,27 @@ public static JsonObject getVCAPServices() {
...
+            JsonObject credentials = vcapObject.get("credentials").getAsJsonObject();
+            Map<String,String> map = new HashMap<String,String>();
+            Set<Map.Entry<String, JsonElement>> entrySet = credentials.entrySet();
+            for(Map.Entry<String, JsonElement> entry : entrySet) {
+                map.put(entry.getKey(), credentials.get(entry.getKey()).getAsString());
+            }
+
+            return map;
+        } catch (Throwable t) {
+            System.out.println("faild to parse VCAP" + t);
```
Comment (sjfink) [inline tests/src/common/TestUtils.java]: faild -> failed

File: tests/src/common/TestUtils.java [hunk 2]
```diff
@@ -103,6 +108,27 @@ public static JsonObject getVCAPServices() {
...
+     * @return VCAP credentials as a <String, String> map for each <property, value> pair in credentials
+     */
+    public static Map<String, String> getVCAPcredentials(String vcapService) {
+        try {
+            JsonArray vcapArray = getVCAPServices().get(vcapService).getAsJsonArray();
+            JsonObject vcapObject = vcapArray.get(0).getAsJsonObject();
+            JsonObject credentials = vcapObject.get("credentials").getAsJsonObject();
+            Map<String,String> map = new HashMap<String,String>();
+            Set<Map.Entry<String, JsonElement>> entrySet = credentials.entrySet();
+            for(Map.Entry<String, JsonElement> entry : entrySet) {
```
Comment (sjfink) [inline tests/src/common/TestUtils.java]: could simply iterate over credentials.keySet()

**Comment by nwspeete-ibm** [comment]
[READY]

**Comment by sjfink** [comment]
rebased and merged

**Pull request closed without merging**